app/AppKernel.php

```diff
@@ -37,11 +37,15 @@ public function registerBundles()
             new FOS\RestBundle\FOSRestBundle(),
             new JMS\SerializerBundle\JMSSerializerBundle(),
 
+            // Application bundles
+            new AppBundle\AppBundle(),
+
             // Remove this Bundle when using the SE as the basis for a new project
             new Acme\DemoBundle\AcmeDemoBundle(),
```

Comment:
I don't think so. The AppBundle should be kept clean and a seperate demo bundle makes it easier to remove it. This is also how the core SE does it